playbooks/empty-dir-quota.yaml

```diff
@@ -0,0 +1,7 @@
+- hosts: schedulable_nodes
+  gather_facts: yes
+  become: yes
+  vars_files:
+    - vars/main.yaml
```

Comment:
vars_files may be problematic, I would suggest that required settings be passed in for generic playbooks.